docs/integrations/dbt.md

```diff
@@ -214,6 +214,25 @@ seeds:
         data_type: <SQL data type>
 ```
 
+Alternatively, you can specify a [column_types](https://docs.getdbt.com/reference/resource-configs/column_types) dictionary in your `dbt_project.yml` file, where the keys define the column names and the values the data types.
```

Comment:
I think this should actually be a primary way to specify column types for seeds

Reply:
Right realigned this part